solr/core/src/java/org/apache/solr/cloud/api/collections/CreateCollectionCmd.java

```diff
@@ -264,8 +299,8 @@ public void call(ClusterState clusterState, ZkNodeProps message, @SuppressWarnin
         log.info("Cleaned up artifacts for failed create collection for [{}]", collectionName);
         throw new SolrException(ErrorCode.BAD_REQUEST, "Underlying core creation failed while creating collection: " + collectionName);
       } else {
-        log.debug("Finished create command on all shards for collection: {}", collectionName);
 
+        log.debug("Finished create command on all shards for collection: {}", collectionName);
```

Comment:
When we get here for a PRS collection, the `ZkStateReader` cluster state for the collection is fine, as are the ZK structures, but the Overseer cluster state updater does not know about the collection. Any new operation on the collection through the Collection API would fail in the cluster state updater.